src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -82,9 +83,14 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
             return {it->second};
         }
     } else if (name == ov::supported_properties.name()) {
-        return std::vector<ov::PropertyName>{
-            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
-            ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO}};
+        // return supported_configKeys;
+        std::vector<ov::PropertyName> property_name;
+        property_name.push_back(ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO});
+        property_name.push_back(ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO});
```

Comment:
Should we register the property `ov::supported_properties` and `ov::device::full_name` into `supported_configKeys` in initiation step?

Reply:
resolve this comment as this is the original code logic.